java/pom.xml

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Any reason why you deleted this file?

Reply:
We are not providing this solution as maven artifact, This is simple java source code library which can added into end user project. This comment was given by Louis. Hence made this change.

Reply:
That is not true. We need to provide the library on public repositories, in this case on maven.org. For that reason the OSS team has added the pom file, even if we are not quite ready to release it yet. Please restore it.

Reply:
Thank you! Lene. I have restored the pom.xml.